src/main.rs

```diff
@@ -24,8 +24,8 @@ fn main(req: Request) -> Result<Response, Error> {
         // Allow GET and HEAD requests.
         &Method::GET | &Method::HEAD => (),
 
-        // Deny anything else.
-        _ => {
+        // Deny anything other than PURGE requests.
+        _ => if req.get_method_str() != "PURGE" {
```

Comment:
Above, you have a condition that checks for allowed methods, and this effectively does the same thing, so I wonder whether it would be simpler to write line 25 as `&Method::GET | &Method::HEAD | &Method::PURGE` ?

Reply:
It would be simpler to write the line that way, but "PURGE" is not part of the [Method struct](https://docs.rs/fastly/latest/fastly/http/struct.Method.html) in the Rust crate. I believe we're going to talk about it in the next ECP sync.